### PR TITLE
Update consts.ts

### PR DESCRIPTION
**Pull request by bitcoin21ideas**
Added tags and emojis for cocktails preparation methods. Will help readers choose cocktails based on the tools, experience and time they have.

### CODE REVIEW

**Comment by github-tijlxyz** [comment]
Thanks for the PR.
Few things; think the emoji usage is a bit inconsistent, think we should remove a few.

File: src/lib/consts.ts [hunk 1]
```diff
@@ -37,10 +37,12 @@ export const recipeTags: recipeTagSimple[] = [
   { title: 'Beef', emoji: '🐄' },
   { title: 'Beetroot' },
   { title: 'Belgian', emoji: '🇧🇪' },
+  { title: 'Blended', emoji: '💫' },
```
Comment (github-tijlxyz) [inline src/lib/consts.ts]:
```suggestion
  { title: 'Blended' },
```

File: src/lib/consts.ts [hunk 2]
```diff
@@ -37,10 +37,12 @@ export const recipeTags: recipeTagSimple[] = [
...
   { title: 'Brazilian', emoji: '🇧🇷' },
   { title: 'Bread', emoji: '🍞' },
   { title: 'Breakfast', emoji: '🍳' },
   { title: 'Broccoli' },
+  { title: 'Built', emoji: '🧱' },
```
Comment (github-tijlxyz) [inline src/lib/consts.ts]:
```suggestion
  { title: 'Built' },
```

File: src/lib/consts.ts [hunk 3]
```diff
@@ -90,6 +93,7 @@ export const recipeTags: recipeTagSimple[] = [
   { title: 'Japanese', emoji: '🇯🇵' },
   { title: 'Keto' },
   { title: 'Lamb', emoji: '🐑' },
+  { title: 'Layered', emoji: '💈' },
```
Comment (github-tijlxyz) [inline src/lib/consts.ts]:
```suggestion
  { title: 'Layered' },
```

**Review by github-tijlxyz — CHANGES_REQUESTED** [review]

**Comment by bitcoin21ideas** [comment]
Sure. Totally your call. I just added the ones that I found closest to the maening.

**Review by github-tijlxyz — APPROVED** [review]

**Pull request merged**